tests/setup-runtime-dependencies.cmd

```diff
@@ -42,22 +42,29 @@ REM ============================================================================
 set __DotNetToolDir=%__ThisScriptPath%..\Tools
 set __DotNetCmd=%__DotNetToolDir%\dotnetcli\bin\dotnet.exe
 set __PackageDir=%__ThisScriptPath%..\Packages
-set __JasonFilePath=%__ThisScriptPath%project.json
+set __TmpDir=%Tmp%\coreclr_gcstress_%RANDOM%
 
-REM Check if dotnet CLI exists
+REM Check if donet cli exists
 if not exist "%__DotNetToolDir%" (
     echo Directory containing dotnet CLI does not exist: %__DotNetToolDir%
-    exit /b 1
+    goto Fail
 )
 if not exist "%__DotNetCmd%" (
     echo dotnet.exe does not exist: %__DotNetCmd%
-    exit /b 1
+    goto Fail
 )
 
 REM Create directories needed
 if not exist "%__PackageDir%" md "%__PackageDir%"
 if not exist "%__OutputDir%" md "%__OutputDir%"
 
+REM Check Tmp directory
+if not exist "%__TmpDir%" (
+    md %__TmpDir%
+)
```

Comment:
In case __TmpDirs exists (somehow we accidently didn't delete it in the prior run or somehow this directory is made before), we should delete it. And then create it always.